src/submission/submission-service.ts

```diff
@@ -24,18 +24,24 @@ import {
   ClinicalEntities,
   ClinicalSubmissionModifierCommand,
   RegistrationStat,
+  NewClinicalEntity,
+  SubmissionBatchError,
+  SubmissionBatchErrorTypes,
+  ValidateSubmissionResult,
+  NewClinicalEntities,
 } from './submission-entities';
 import * as schemaManager from '../lectern-client/schema-manager';
 import {
   SchemaValidationError,
   TypedDataRecord,
   DataRecord,
   SchemaProcessingResult,
+  SchemaValidationErrorTypes,
 } from '../lectern-client/schema-entities';
 import { loggerFor } from '../logger';
-import { Errors, F } from '../utils';
+import { Errors, F, isStringMatchRegex, TsvUtils } from '../utils';
 import { DeepReadonly } from 'deep-freeze';
-import { FileType } from './submission-api';
+import { FileType, FileNameRegex } from './submission-api';
```

Comment:
FileNameRegex, move to the submission-service.ts and rename to BatchNameRegex

Reply:
the service shouldn't use anything or have any reference on the API, FileType should be moved too, and renamed to ClinicalSubmissionEntity or something more domain related